### ISSUE TITLE
missing tag error

### ISSUE THREAD

**Issue by mmwwah** [scottkleinman/aeme#78]
I'm trying to switch into grid view, for my sanity.  But oXygen won't let me, claiming that "The element `<l>` must be terminated by the matching end-tag `</l>`.  The only problem is that the end-tag is there in the code.  I've tried deleting and retyping both the opening and closing tags, to no avail.  The answer has got to be obvious but I'm just missing it.  Help a sister out?

Here is the full verse line.  Also, the error is for code line 39, right in the middle of the verse line.  

![screen shot 2015-11-13 at 09 26 57](https://cloud.githubusercontent.com/assets/7809545/11148196/47228bb0-89e9-11e5-874b-d8198ebfd3d3.png)

**Comment by scottkleinman** [comment]
You have an extra `</name>` end tag in line 33. Delete that and see if the error goes away.

**Comment by mmwwah** [comment]
Doh, thanks.  That was it.  A few other open tags throughout the Dedication threw `</l>` errors (none of them actually line tags), and I fixed them.  Now the final one.  What am I missing?
```            <l xml:id="junius1.292">me &thorn;i&slong;&slong; bede biddenn<pc>&middot;</pc></l>
            <l xml:id="junius1.293"><pc>&parag;</pc> &thorn;att bro&thorn;err &thorn;att</l>
            <l xml:id="junius1.294">ti&slong;&slong; enngli&slong;&slong;h &wynn;ritt</l>
            <l xml:id="junius1.295">Allr &aelig;re&slong;&slong;t &wynn;ra&dblac;t</l>
            <l xml:id="junius1.296">&amp; &wynn;rohhte<pc>&punctelev;</pc> &thornbar;</l>
            <l xml:id="junius1.297">bro&thorn;err for<choice>
                <orig>&rsup;</orig>
                <reg>r</reg>
             </choice></l>
            <l xml:id="junius1.298">hi&slong;&slong; &slong;&wynn;i&dblac;c to</l>
            <l xml:id="junius1.299">l&aelig;n<pc>&punctelev;</pc> So&thorn;</l>
            <l xml:id="junius1.300">bli&slong;&slong;e mo&#combacute;te</l>
               
         </div1>
      </body>
  </text>
</TEI>```

The error code is attached to the `</div1>`.  I have gone back and checked for other `<div>`s and `</div>`s, and there's nothing.

**Comment by scottkleinman** [comment]
Have you checked that your code begins like this?

```xml
<text>
  <body>
    <div1>
```

**Comment by mmwwah** [comment]
Yep.  Is the naming of the MS in `<body>` throwing it off, perhaps?

![screen shot 2015-11-13 at 12 09 53](https://cloud.githubusercontent.com/assets/7809545/11152271/988561f6-89ff-11e5-939b-962979053964.png)

**Comment by mmwwah** [comment]
The placement of the comments shouldn't foul the waters, should it?

**Comment by scottkleinman** [comment]
No, all of that should be good. Whatever is causing the problem is not something that will be detected easily with a small fragment of code. If you want to send me the whole thing, I'll have a look.

**Comment by mmwwah** [comment]
Okay, I'll push everything to meg-dedication -- or would you rather me email you the file?  

Unresolved tag notwithstanding, any thoughts you have re the first 350 lines of code (the part I've gone over) would be quite welcome.  I've tried to automate as much as I can, but it's still slow going.  And I may well be screwing something up.  Not that I'm asking you to do a close code review -- just if anything jumps out at you.

Thanks!

**Comment by scottkleinman** [comment]
Might as well push it. I have a meeting-heavy day, but I'll try to look at it during breaks.

**Comment by mmwwah** [comment]
No worries if you don't get to it.  Happy meeting-ing...
